static/app/views/settings/account/notifications/utils.tsx

```diff
@@ -385,8 +384,6 @@ export const getParentField = (
     parentId: string
   ) => NotificationSettingsObject
 ): FieldObject => {
-  /** Render each parent and add a default option to the the field choices. */
```

Comment:
Can we auto format this?

Reply:
What are you thinking?

Reply:
idk if there's a lint rule for it

Reply:
Like no comments like `/** ... */`?

Or also auto wrapping?

Reply:
Comment style